google/cloud/BUILD.bazel

```diff
@@ -67,6 +67,12 @@ config_setting(
     },
 )
 
+filegroup(
+    name = "common_hdrs",
+    srcs = [h for h in google_cloud_cpp_common_hdrs if not h.startswith("internal/")],
```

Comment:
Observation: we are doing this `internal` filtering on handwritten libraries. Our fully generated libraries also have `internal` headers which are picked up by globs.

Do we care to filter both?

Reply:
Yes.  That is one of the reasons why this is "part of" and not "fixes" for that bug.

Reply:
I included the fixes in this PR. PTAL